### PR TITLE
Restrict user from claiming a pull request.

### PR DESCRIPTION
**Pull request by sayyedarib**
[CZO thread](https://chat.zulip.org/#narrow/stream/105-zulipbot/topic/claim.20pull.20request/near/1711447)

| Before | After |
| --- | --- |
|Screenshot from 2024-01-04 14-28-15| Screenshot from 2024-01-08 13-22-23|

### CODE REVIEW

File: config/templates/claimPullRequest.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Hello @{commenter}!
+
+Thanks for your interest in Zulip! Please note that claiming pull requests is not supported. You can exclusively claim and submit pull requests for issues bearing the [help wanted](https://github.com/{repoOwner}/{repoName}/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee+label%3A%22help+wanted%22) label.
+
+If this is your first time here, we recommend reading our [guide for new contributors](https://zulip.readthedocs.io/en/latest/overview/contributing.html) before getting started.
```
Comment (alya) [inline config/templates/claimPullRequest.md:5]: Let's make it:

---

Hello @{commenter}!

You have attempted to claim a pull request, which is someone else's proposed solution for an issue. Please review our [guide for new contributors](https://zulip.readthedocs.io/en/latest/overview/contributing.html) to learn how to find an issue to work on.

**Comment by sayyedarib** [comment]
I've updated the texts per your suggestions. Please take another look and let me know if any other changes are needed.

**Comment by alya** [comment]
Thanks! @andersk are you the right person to review this PR?

**Comment by alya** [comment]
@timabbott are you up for reviewing?

**Comment by andersk** [comment]
The first commit will crash without the second, so it is not a [coherent logical unit](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#each-commit-must-be-coherent). The two commits need to be squashed together.

**Comment by andersk** [comment]
Thanks @sayyedarib, this is live now.

**Pull request merged**